pkg/client/statistics.go

```diff
@@ -0,0 +1,34 @@
+package client
+
+import "context"
+
+type Statistics struct {
+	DocumentsTotal         int64              `json:"documents_total"`
+	DocumentsInbox         int64              `json:"documents_inbox"`
+	InboxTag               int64              `json:"inbox_tag"`
+	InboxTags              []int64            `json:"inbox_tags"`
+	DocumentFileTypeCounts []DocumentFileType `json:"document_file_type_counts"`
+	CharacterCount         int64              `json:"character_count"`
+	TagCount               int64              `json:"tag_count"`
+	CorrespondentCount     int64              `json:"correspondent_count"`
+	DocumentTypeCount      int64              `json:"document_type_count"`
+	StoragePathCount       int64              `json:"storage_path_count"`
+	CurrentAsn             int64              `json:"current_asn"`
+}
+
+type DocumentFileType struct {
```

Comment:
Please rename to `StatisticsDocumentFileType`. I know the name follows the upstream naming, but the "type" reads as if the structure were a type only when it's actually the combination of a type and a count.

Reply:
https://github.com/hansmi/paperhooks/pull/43/commits/420ca1e3a18563a016cc00ed20f2d4cae00fa18d

Reply:
Fixed in https://github.com/hansmi/paperhooks/pull/43/commits/420ca1e3a18563a016cc00ed20f2d4cae00fa18d